gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Sat Aug 22 19:36:10 CEST 2015
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
```

Comment:
Adding the following line allows the gradle wrapper to verfiy the downloaded gradle zip:
`distributionSha256Sum=9af7345c199f1731c187c96d3fe3d31f5405192a42046bafa71d846c3d9adacb`

Reply:
Oh, good to know. I added the line. Thanks!